graphql/subcourse/fields.ts

```diff
@@ -438,13 +438,31 @@ export class ExtendedFieldsSubcourseResolver {
         return await canPublish(subcourse);
     }
 
+    @FieldResolver((returns) => Decision)
+    @Authorized(Role.ADMIN, Role.OWNER)
+    async canCancel(@Root() subcourse: Required<Subcourse>) {
+        return await canCancel(subcourse);
+    }
+
+    @FieldResolver((returns) => Decision)
+    @Authorized(Role.ADMIN, Role.OWNER)
+    async canEdit(@Root() subcourse: Required<Subcourse>) {
+        return await canEditSubcourse(subcourse);
+    }
+
     @FieldResolver((returns) => Decision)
     @Authorized(Role.PARTICIPANT, Role.INSTRUCTOR)
     async canContactInstructor(@Root() subcourse: Required<Subcourse>) {
         const course = await getCourse(subcourse.courseId);
         return await canContactInstructors(course, subcourse);
     }
 
+    @FieldResolver((returns) => Decision)
+    @Authorized(Role.PARTICIPANT, Role.INSTRUCTOR)
+    async canContactParticipants(@Root() subcourse: Required<Subcourse>) {
```

Comment:
Can a participant contact another participant?

Reply:
No, but they might want to know whether course instructors could contact them (hypothetically). I'm usually very lose with access restrictions on the can... Resolvers as on some pages it might make sense to do the same query for different types of users without having to do different queries for different roles. 
I guess canInstructorContactParticipants would be a better name, but we already have canContactInstructors so that is kind of symmetrical

Reply:
Okay, I see 👍